packages/components/src/internal/components/lineage/node/LineageDetail.tsx

```diff
@@ -1,22 +1,88 @@
 import React, { FC, memo, useMemo } from 'react';
 import { Experiment, Filter } from '@labkey/api';
+import { Map } from 'immutable';
 
-import { DetailPanelWithModel, QueryConfig, SchemaQuery } from '../../../..';
+import { DetailPanel, LoadingSpinner, QueryColumn, resolveDetailRenderer, SchemaQuery } from '../../../..';
+
+import { InjectedQueryModels, QueryConfigMap, withQueryModels } from '../../../../public/QueryModel/withQueryModels';
+
+const ADDITIONAL_DETAIL_FIELDS = ['properties'];
 
 export interface LineageDetailProps {
     item: Experiment.LineageItemBase;
 }
 
+const LineageDetailImpl: FC<LineageDetailProps & InjectedQueryModels> = memo(props => {
+    const { actions, queryModels } = props;
+    if (queryModels.model.isLoading) return <LoadingSpinner />;
+
+    const additionalCols = queryModels.model.allColumns.filter(
+        col => ADDITIONAL_DETAIL_FIELDS.indexOf(col.fieldKey?.toLowerCase()) > -1
+    );
+    const detailColumns = [...queryModels.model.detailColumns, ...additionalCols];
+
+    return (
+        <DetailPanel
+            model={queryModels.model}
+            actions={actions}
+            queryColumns={detailColumns}
+            detailRenderer={_resolveDetailRenderer}
+        />
+    );
+});
+
+const LineageDetailWithQueryModels = withQueryModels<LineageDetailProps>(LineageDetailImpl);
+
 export const LineageDetail: FC<LineageDetailProps> = memo(({ item }) => {
-    const queryConfig: QueryConfig = useMemo(
+    const queryConfigs = useMemo<QueryConfigMap>(
         () => ({
-            baseFilters: item.pkFilters.map(pkFilter => Filter.create(pkFilter.fieldKey, pkFilter.value)),
-            containerPath: item.container,
-            schemaQuery: SchemaQuery.create(item.schemaName, item.queryName),
+            model: {
+                baseFilters: item.pkFilters.map(pkFilter => Filter.create(pkFilter.fieldKey, pkFilter.value)),
+                containerPath: item.container,
+                schemaQuery: SchemaQuery.create(item.schemaName, item.queryName),
+                requiredColumns: ['*'],
+            },
         }),
         [item]
     );
 
-    // Without providing "key" the DetailPanelWithModel will stop updates
-    return <DetailPanelWithModel key={item.lsid} queryConfig={queryConfig} />;
+    // providing "key" to allow for reload on lsid change
+    return <LineageDetailWithQueryModels key={item.lsid} autoLoad queryConfigs={queryConfigs} item={item} />;
+});
+
+interface RendererProps {
+    data: Map<string, any>;
+}
+
+// exported for jest testing
+export const CustomPropertiesRenderer: FC<RendererProps> = memo(props => {
+    const { data } = props;
+
+    return (
+        <table className="lineage-detail-prop-table">
+            <tbody>
+                {data.map(row => {
+                    const fieldKey = row.get('fieldKey');
+                    const name = fieldKey.substring(fieldKey.indexOf('#') + 1);
```

Comment:
I guess I would expect the query metadata or queryInfo to give us more information so we don't have to parse the fieldKey to find out. I thought that Kevin had taken steps to add this to the query metadata returned from selectRows. If that is not made readily available it may be getting pulled through from the `QueryInfo` correctly. This would be the same column information you're filtering for above when you utilized the `ADDITIONAL_DETAIL_FIELDS`. The `col` should be made available via the render props.